src/runtime_src/tools/xclbinutil/CMakeLists.txt

```diff
@@ -136,6 +136,11 @@ if(NOT WIN32)
    target_link_libraries(${XCLBINUTIL_NAME} PRIVATE transformcdo)
 endif()
 
+# Link required windows specific libraries
+if(WIN32)
+  target_link_libraries(${XCLBINUTIL_NAME} PRIVATE wsock32 ws2_32)
```

Comment:
Not sure why these link libraries are needed?

Reply:
winsock is included i.e. here and on other places in xclbinutil: https://github.com/Xilinx/XRT/blob/c34c5afb4c92cb68460349185da7e8fb24f9d957/src/runtime_src/tools/xclbinutil/DTC.cxx#L26
ws2_32 is included here and used by some of the binaries: https://github.com/Xilinx/XRT/blob/c34c5afb4c92cb68460349185da7e8fb24f9d957/src/runtime_src/core/tools/common/XBUtilities.cpp#L32